comps/cores/mega/gateway.py

```diff
@@ -211,6 +211,7 @@ async def handle_request(self, request: Request):
         prompt = self._handle_message(chat_request.messages)
         print("prompt in gateway", prompt)
         parameters = LLMParams(
+            model=chat_request.model,
```

Comment:
This change is already added below. Please refer to line number 224. I think more that this we can say on line number 224 as below:
model=chat_request.model if chat_request.model else (getenv(MODEL_ID) if getenv(MODEL_ID) else Neuralchat)